front-end/src/components/Post/PostGovernanceInfo/PostTreasuryInfo.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { Grid } from 'semantic-ui-react';
+import styled from '@xstyled/styled-components';
+
+import AddressComponent from '../../../ui-components/Address';
+import { OnchainLinkTreasuryProposalFragment } from '../../../generated/graphql';
+import { chainProperties } from '../../../global/networkConstants';
+import getNetwork from '../../../util/getNetwork';
+
+interface Props{
+	className?: string
+	onchainLink: OnchainLinkTreasuryProposalFragment
+}
+
+const currentNetwork = getNetwork();
+
+const PostTreasuryInfo = ({ className, onchainLink }: Props) => {
+	if (!onchainLink) return null;
+
+	const {
+		onchain_treasury_spend_proposal: onchainTreasuryProposal,
+		proposer_address: proposerAddress
+	} = onchainLink;
+	const beneficiary = onchainTreasuryProposal?.[0]?.beneficiary;
+	const value = onchainTreasuryProposal?.[0]?.value;
+	const bond = onchainTreasuryProposal?.[0]?.bond;
+
+	return (
+		<div className={className}>
+			<h4>On-Chain Info</h4>
```

Comment:
This can be wrapped in the `OnchainInfoWrapper` now instead of a div with the `className` variable. Can get rid of `className` var, css styling and styled-components import then.

Reply:
done